scripts/release.sh

```diff
@@ -49,8 +49,8 @@ if [[ $PRERELEASE == "" ]]; then
     [[ $FIRST == 'Y' || $FIRST == 'y' ]] && PRERELEASE=true
 fi
 
-sed -i '' "s|VERSION=\".*\"|VERSION=\"${RELEASE_TAG}\"|" scripts/install.sh
-sed -i '' "s|ENV VERSION .*|ENV VERSION ${RELEASE_TAG}|" Dockerfile
```

Comment:
IIRC, the `-i ''` is so it works on certain systems... I don't think any OS will have issues with the `''` being there, but some will if it's missing, so I think we want to leave them.